### DISCUSSION TITLE
[Error] The remote server returned an error: (403) Forbidden

### DISCUSSION THREAD

**Discussion by Scorpias1** [Zerg00s/FlowPowerAppsMigrator#65] [Q&A] [answered]
I've been using this app successfully for 2 years.  Today it stopped working.  All I get is the following error:

![image](https://github.com/Zerg00s/FlowPowerAppsMigrator/assets/143567905/1161325c-0db3-4631-8624-bbc7b6a32675)

-I absolutely have Site Collection Admin access
-I have re-installed the app.
-I have tried a different computer
-I have used Clear Credential Cache = true
-I am not a SharePoint Online tenant admin.  I cannot create an application in Azure AD.
-I signed in with Internet Explorer
-I successfully tested my connection to SP in PowerShell using this:
  `   Install-Module SharePointPnPPowerShellOnline -Scope CurrentUser
     Connect-PnPOnline -Url <Your SharePoint site> -UseWebLogin -WarningAction Ignore
     Get-PnPList -Includes Views, Fields, DefaultView`

What else can I try?

**Comment by Zerg00s** [comment]
Hi @Scorpias1. Thank you for the details and a screenshot. That's super useful. 
Looks like, for some reason, a REST API call fails with a 403 error. I will create a new release that will not use REST API. Can you try this new release for me? Stay tuned.

**Comment by Zerg00s** [comment, marked as answer]
@Scorpias1 , can you try this fresh release, please? I hope, that will fix this 403 error:

https://github.com/Zerg00s/FlowPowerAppsMigrator/releases/tag/3.9.5

**Reply by Scorpias1** [reply]
That fixed it!  Thank you so much!!!
